src/api.ts

```diff
@@ -7,311 +7,311 @@
  * Copy the `api.ts` source file into your project to use it.
  */ /** */
 
-import {DebugSession, Disposable, Event, Terminal} from 'vscode';
+import { DebugSession, Disposable, Event, Terminal } from 'vscode';
```

Comment:
It looks like there is some other formatter changing the spacing on `{}` blocks. I don't think we want to do that as part of this change.

Reply:
Maybe add `"javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false` to settings.json

Reply:
It seems that breaks the linter for single-line code blocks since it wants whitespace after a semicolon. :(

Reply:
But we could do a find/replace on `;}` and add a space.  Or just leave this for now and fix it when we set up a formatter. The built-in Typescript formatter just doesn't have enough options unfortunately.